ngraph_onnx/onnx_importer/backend.py

```diff
@@ -79,7 +127,7 @@ def __init__(self, ng_model, device='CPU'):  # type: (List[Dict], str) -> None
         super(NgraphBackendRep, self).__init__()
```

Comment:
The `device` argument to this constructor may be an ONNX value such as `CUDA`. Please add support for ONNX device names here.